From 354a5bd0f11a46a9e4229072b41ee321a9ed5e21 Mon Sep 17 00:00:00 2001
From: Christopher Degawa <ccom@randomderp.com>
Date: Fri, 18 Sep 2020 19:16:34 +0000
Subject: [PATCH 03/12] modules/access/srt: Use srt_create_socket instead of
 srt_socket

fixes

```c
../../modules/access_output/srt.c: In function 'srt_schedule_reconnect':
../../modules/access_output/srt.c:119:5: warning: 'srt_socket' is deprecated [-Wdeprecated-declarations]
  119 |     p_sys->sock = srt_socket( res->ai_family, SOCK_DGRAM, 0 );
      |     ^~~~~
In file included from ../../modules/access/srt_common.h:31,
                 from ../../modules/access_output/srt.c:25:
D:/media-autobuild_suite/local32/include/srt/srt.h:770:41: note: declared here
  770 | SRT_ATR_DEPRECATED_PX SRT_API SRTSOCKET srt_socket(int, int, int) SRT_ATR_DEPRECATED;
      |                                         ^~~~~~~~~~
```

Signed-off-by: Christopher Degawa <ccom@randomderp.com>
---
 modules/access/srt.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/modules/access/srt.c b/modules/access/srt.c
index d7ef51fccf..5d8597ce4c 100644
--- a/modules/access/srt.c
+++ b/modules/access/srt.c
@@ -125,7 +125,7 @@ static bool srt_schedule_reconnect(stream_t *p_stream)
         srt_close( p_sys->sock );
     }
 
-    p_sys->sock = srt_socket( res->ai_family, SOCK_DGRAM, 0 );
+    p_sys->sock = srt_create_socket();
     if ( p_sys->sock == SRT_INVALID_SOCK )
     {
         msg_Err( p_stream, "Failed to open socket." );
-- 
2.25.1

